src/stores/AuthContext.jsx

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import { createContext, useContext, useState } from 'react';
+
+import { API_ENDPOINT } from '../config/config.jsx';
+
+const AuthContext = createContext({
+  users: { indetifier: '', password: '' },
+  // isLoggedIn: false,
+  login: () => {},
+  handleLogout: () => {},
+});
+
+export const useAuthContext = () => useContext(AuthContext);
+
+export const AuthContextProvider = ({ children }) => {
+  const [user, setUser] = useState({});
+  // const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const login = async (e, indetifier, password, resetEmailInput, resetPasswordInput) => {
```

Comment:
The login function in this context should not get the `e`, `resetEmailInput` and `resetPasswordInput` as params - it should not know about these things, the point of this function should be to
1. Get email and password
2. Do the login call
3. Set jwt and user state variables

It should not know anything about the outside components

Also, typo, `indetifier` => `identifier`